app/routes/circulars/index.tsx

```diff
@@ -191,7 +196,21 @@ export default function () {
 
   return (
     <>
-      <h1>GCN Circulars</h1>
+      <Grid row>
+        <div className="tablet:grid-col flex-fill">
+          <h1 className="float-left margin-bottom-0">GCN Circulars</h1>
+        </div>
+        <div className="tablet:grid-col flex-auto">
+          <Link to={`/circulars/new${searchString}`}>
+            <Button
+              type="button"
+              className="height-4 padding-top-0 padding-bottom-0"
+            >
+              <Icon.Edit /> Submit New Circular
+            </Button>
+          </Link>
+        </div>
+      </Grid>
```

Comment:
These UX changes appear to be unrelated to the number of results.

Reply:
> These UX changes appear to be unrelated to the number of results.

![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/57914086/b6d40f84-b13c-498a-b6f7-352f4f818316)

This was a change made when I had originally placed the result selector dropdown inline with the search box, but it's not entirely necessary now that the result selector is at the bottom of the page. My rationale was that repositioning the new circular button would reduce the business of the search bar/make it so controls inline with the search bar were only those related to searching. Moving it to the top of the page also made it a bit quicker to find for those only interested in submitting a new circular

Reply:
It's a reasonable idea, but it's important to keep each PR focused on _one_ change, bug fix, or feature. If you want to pursue this design change, then it needs to be in a separate PR.

Reply:
ah, yeah, that's a fair point. In order to revert it to have the same appearance as before and also pull the Form element out of the button group, I had to do a nested button group type structure in order for the search and new button to have the same appearance as before the form and button group element order had to be changed.

Reply:
moved this to a separate pr (#1142)